Sources/SentryCrash/Recording/Tools/SentryCrashObjCApple.h

```diff
@@ -288,8 +288,8 @@ typedef struct __CFRuntimeBase {
 #    define __CF_BIG_ENDIAN__ 0
 #endif
 
-#define CF_INFO_BITS (!!(__CF_BIG_ENDIAN__)*3)
-#define CF_RC_BITS (!!(__CF_LITTLE_ENDIAN__)*3)
+#define CF_INFO_BITS (!!(__CF_BIG_ENDIAN__) * 3)
```

Comment:
Can we avoid this file being formatted and included in the commit?

Reply:
Not sure, I did not change this. Probably the formatter.